sources/configure.ml

```diff
@@ -15,6 +15,7 @@
 let prefix = ref ""
 let libdir = ref ""
 let mandir = ref ""
+let static = ref false
```

Comment:
I think it'd be better to have a variant `type profile = Dev | Release | Static` instead of a simple boolean (among other things to correctly allow to build in dev mode, see the other comment).